docker/.env.common

```
@@ -32,3 +33,15 @@ IO_BACKEND_URL=https://localhost:8000
 FF_UNIQUE_EMAIL_ENFORCEMENT="ALL"
 FF_UNIQUE_EMAIL_ENFORCEMENT_USERS=["ISPXNB32R82Y766D"]
 PROFILE_EMAIL_STORAGE_TABLE_NAME="uniqueEmails"
+
+
```

please add APIM mocked required groups. if u notice this env var is present in `docker-compose.yml` file
```suggestion
REQ_USER_GROUPS="ApiLollipopAssertionRead"
```